hansemannn/titanium-firebase-core
# Android: Add file parameter

Thanks to @appwerft for the ground work (https://github.com/hansemannn/titanium-firebase-core/issues/11)

**Test setup:**
* rename the google-services.json to gs.json
* place the file in `app/assets/android/gs.json`
* check the debug log for the `Firebase Options`
```
var fc = require('firebase.core');
fc.configure({
	file: "gs.json"
});
```

Debug message will also appear when you use the other parameter like this:
```
fc.configure({
	APIKey:"1",
	databaseURL:"2",
	projectID:"3",
	storageBucket: "4",
	applicationID: "5",
	GCMSenderID: "6"
});
```
